src/Jackett.Common/Indexers/TorrentBytes.cs

```diff
@@ -143,74 +142,66 @@ private async Task ProcessPage(List<ReleaseInfo> releases, string searchUrl)
                 response = await RequestStringWithCookiesAndRetry(searchUrl, null, BrowseUrl);
             }
 
-            var results = response.Content;
             try
             {
-                CQ dom = results;
-
-                var rows = dom["table > tbody:has(tr > td.colhead) > tr:not(:has(td.colhead))"];
+                var parser = new HtmlParser();
+                var dom = parser.ParseDocument(response.Content);
+                var rows = dom.QuerySelectorAll("table > tbody:has(tr > td.colhead) > tr:not(:has(td.colhead))");
                 foreach (var row in rows)
                 {
                     var release = new ReleaseInfo();
 
-                    var link = row.Cq().Find("td:eq(1) a:eq(1)").First();
-                    release.Guid = new Uri(SiteLink + link.Attr("href"));
+                    var link = row.QuerySelector("td:nth-of-type(2) a:nth-of-type(2)");
+                    release.Guid = new Uri(SiteLink + link.GetAttribute("href"));
                     release.Comments = release.Guid;
-                    release.Title = link.Attr("title");
+                    release.Title = link.GetAttribute("title");
 
                     // There isn't a title attribute if the release name isn't truncated.
                     if (string.IsNullOrWhiteSpace(release.Title))
-                    {
-                        release.Title = link.Get(0).FirstChild.ToString().Trim();
-                    }
+                        release.Title = link.FirstChild.ToString().Trim();
```

Comment:
I'm not sure here.

Reply:
csQuery is supposed to be a functional clone of jQuery. Here's the documentation of `.Get()` https://api.jquery.com/get/ which is the same as [index] in C#. This is correct.